x-pack/legacy/plugins/siem/public/components/events_viewer/mock.ts

```diff
@@ -43,2062 +44,12 @@ export const mockEventViewerResponse = [
           id: 'default',
           Timeline: {
             totalCount: 12,
-            inspect: null,
             pageInfo: {
-              endCursor: {
-                value: '1566857636197',
-                tiebreaker: '19378194',
-                __typename: 'CursorType',
-              },
+              endCursor: null,
               hasNextPage: true,
               __typename: 'PageInfo',
             },
-            edges: [
```

Comment:
This mock was an unexpected touchpoint after re-generating the GraphQL types. Upon further inspection of the tests that depended on this mock, I determined the data itself wasn't required for the (failing) tests, because they just depended on the values of `totalCount` and `hasNextPage`.  As a result, I deleted the mock data in `edges`, and left the tests intact.